motion/thruster_allocator/include/vortex_allocator/eigen_typedefs.h

```diff
@@ -5,8 +5,8 @@
 
 namespace Eigen
 {
-typedef Eigen::Matrix<double, 6, 6> Matrix6d;
-typedef Eigen::Matrix<double, 6, 1> Vector6d;
+typedef Eigen::Matrix<double, 3, 3> Matrix6d;
+typedef Eigen::Matrix<double, 3, 1> Vector6d;
```

Comment:
```suggestion
typedef Eigen::Matrix<double, 3, 3> Matrix3d;
```

The Matrix3d can be changed, and I think Eigen has it's own Vector3d.